src/main/java/com/michalsydoryk/app/gameengine/Game.java

```diff
@@ -0,0 +1,26 @@
+package com.michalsydoryk.app.gameengine;
+
+import com.michalsydoryk.app.board.Board;
+import com.michalsydoryk.app.boardchecker.BoardChecker;
+
+import java.util.Queue;
+
+public class Game {
+    private final Board board;
+    private final BoardChecker boardChecker;
+    private final Players players;
+    private final PlayersPoints playersPoints;
+    private final PlayersSigns playersSigns;
+
+    Game(Board board, BoardChecker boardChecker, Players players, PlayersPoints playersPoints, PlayersSigns playersSigns) {
```

Comment:
Lot of arguments here, maybe use a builder?

Reply:
I have builder for Game in different class.

Reply:
So use it;)